Sourcery/Generating/Template/Stencil/StencilTemplate.swift

```diff
@@ -85,6 +92,15 @@ extension Annotated {
 }
 
 extension Stencil.Extension {
+    
+    func registerFilterWithTwoArguments<A, B>(_ name: String, filter: @escaping (Any?, A, B) throws -> Any?) {
+        registerFilter(name) { (any, args) throws -> Any? in
+            guard args.count == 3, let argA = args[0] as? A, let argB = args[2] as? B else {
+                throw TemplateSyntaxError("'\(name)' filter takes two arguments: \(A.self) and \(B.self)")
+            }
+            return try filter(any, argA, argB)
+        }
+    }
```

Comment:
Good idea @krzysztofzablocki I just amended my commit to 

> Can we reuse registerFilterWithArguments for that, using tuple as arguments type?

Also a good idea, @ilyapuchka. However, I just gave this a shot and came up short :-/